pkg/relayer/session/sessiontree_test.go

```diff
@@ -1,3 +1,96 @@
 package session_test
 
-// TODO: Add tests to the sessionTree logic
+import (
+	"bytes"
+	"compress/gzip"
+	"crypto/rand"
+	"testing"
+
+	"github.com/pokt-network/poktroll/pkg/crypto/protocol"
+	"github.com/pokt-network/smt"
+	"github.com/pokt-network/smt/kvstore/pebble"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSessionTree_CompactProofsAreSmallerThanNonCompactProofs(t *testing.T) {
```

Comment:
#PUC to link to the results: https://github.com/pokt-network/poktroll/pull/823#issuecomment-2363987920